src/shared/sources/_lib/types.js

```diff
@@ -0,0 +1,16 @@
+/** List of allowable crawl types. */
+
+// TODO (techdebt): use this in scraper validation.
+const allowedTypes = [
+  'page',
+  'headless',
+  'csv',
+  'tsv',
+  'pdf',
+  'json',
+  'raw'
+]
+
+module.exports = {
+  allowedTypes
```

Comment:
Why export this as a named param? Is this module going to do more than this?

Reply:
Um ... I don't know! :-)  I thought this was how it had to be done.

Reply:
Leaving for now, can fix later if needed.